src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunWorkflowTests.kt

```diff
@@ -150,14 +150,15 @@ class RunWorkflowTests : SeleniumTest()
     fun `can change branch and see resulting workflow change`()
     {
         createWorkflow()
-        addReport("minimal")
+        changeToOtherBranch()
```

Comment:
Because of the change to the behaviour when getting reports for a branch, this test no longer worked because, the 'other' branch now includes unchanged reports from 'master' branch (global and minimal) - so needed to swap the test round to start creating the workflow on 'other' branch, then switch to 'master' and check that report which does not exist on master has been removed.